cellpack/autopack/Analysis.py

```diff
@@ -2354,6 +2353,8 @@ def pack_one_seed(
                 plt.close()  # closes the current figure
 
         if image_export_options is not None:
+            from cellpack.autopack.writers.ImageWriter import ImageWriter
+
```

Comment:
why did you move this here?

Reply:
I ran a profiler on cellpack to identify time consuming steps and the import of the `ImageWriter` containing `aicsimageio` etc. took up a chunk of time even if images were not actually imported. It's not a very significant time saving so we could move it back to the top to be stylistically consistent.

Reply:
I'm in favor of having imports at the top if it's not going to save very much time